tensorflow/lite/micro/tools/ci_build/test_x86.sh

```diff
@@ -19,50 +19,52 @@
 set -e
 
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-ROOT_DIR=${SCRIPT_DIR}/../../../../..
+if [ ! -z ${1} ]; then
+  ROOT_DIR="${SCRIPT_DIR}/../../../../../.."
+else
+  ROOT_DIR="${SCRIPT_DIR}/../../../../.."
+fi
 cd "${ROOT_DIR}"
 
-source tensorflow/lite/micro/tools/ci_build/helper_functions.sh
+source ${1}tensorflow/lite/micro/tools/ci_build/helper_functions.sh
```

Comment:
let's document the parameters ${1} and $[2} at top of script and assign them to named variables for the rest of the script.

Reply:
Done